zip-1014.rst

```diff
@@ -287,6 +287,16 @@ Grants, but subject to the following additional constraints:
    judges them to violate U.S. law or the ZF's reporting requirements and other 
    (current or future) obligations under U.S. IRS 501(c)(3).
 
+9. A portion of the Discretionary Budget MAY be allocated to provide reasonable 
+   compensation to members of the Major Grant Review Committee. Committee member compensation
+   SHALL be limited to the hours needed to successfully perform their positions
+   and MUST align with the scope and responsibilities of their roles. The
+   allocation and distribution of compensation to committee members SHALL be
+   administered by the ZF. The committee members will receive compensation at a
+   rate of $115 per hour, based on an estimated 15 hours each month, amounting to a
+   monthly sum of $1725. Changes to the hours or rate SHALL be determined by
+   the ZF’s Community Advisory Panel or successor process.
```

Comment:
```suggestion
   compensation to members of the Major Grant Review Committee. The time for which
   each Committee member is compensated SHALL be limited to the hours needed to
   successfully perform their positions, up to a maximum of 15 hours in each month,
   and MUST align with the scope and responsibilities of that member's role. The
   compensation rate for each Committee member SHALL be $115 per hour (and therefore
   the maximum compensation for a Committee member is $1725 per month). The
   allocation and distribution of compensation to committee members SHALL be
   administered by the ZF. Changes to the hours or rate SHALL be determined by
   the ZF’s Community Advisory Panel or successor process.
```

This more closely matches [what was voted on](https://vote.heliosvoting.org/helios/elections/a12e050e-9f1f-11ee-a9a0-6e1a1ab4deb2/view):

> Question `#2`
> Do you support amending ZIP 1014 to provide a stipend of $115 USD per hour (up to a monthly maximum of $1,725 USD) to be paid in ZEC to ZCG Committee members from the Major Grants slice of the Dev Fund?
>
> Yes | 99
> No | 39